libs/community/langchain_community/document_loaders/oracleai.py

```diff
@@ -0,0 +1,460 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2023 - , Oracle and/or its affiliates.
+# -----------------------------------------------------------------------------
+# Authors:
+#   Harichandan Roy (hroy)
+#   David Jiang (ddjiang)
+#
+# -----------------------------------------------------------------------------
+# oracleai.py
+# -----------------------------------------------------------------------------
```

Comment:
can we remove these copyright statements (if you'd like to contribute this code)? all code in this repo is subject to the main license https://github.com/langchain-ai/langchain/blob/master/LICENSE

Reply:
Sure